public/portfolio/index.php

```diff
@@ -39,171 +77,112 @@ function gtag(){dataLayer.push(arguments);}
     		<div class="container-fluid">
 
     			<h1>NOS RÉALISATIONS</h1>
```

Comment:
Il faut harmoniser cette page avec les autres, par exemple le titre :

```suggestion
    			<h1 class="h2 brown-title text-center" style="padding-top:0px;">NOS REALISATIONS</h1>
```